CMake/Servus.cmake

```diff
@@ -0,0 +1,14 @@
+
+set(SERVUS_PACKAGE_VERSION 0.1)
+set(SERVUS_REPO_URL https://github.com/HBPVis/servus.git)
+set(SERVUS_DEPENDS OPTIONAL DNSSD avahi-client Boost)
+set(SERVUS_BOOST_COMPONENTS "unit_test_framework")
+set(SERVUS_DEB_DEPENDS libboost-test-dev avahi-daemon libavahi-client-dev)
+
+if(CI_BUILD_COMMIT)
```

Comment:
I forgot to answer this one. I copy-pasted the cmake file from zeq. I don't understand what this is supposed to do, my best guess is that it's needed by Buildyard+Jenkins in some way, I won't remove it unless I have a confirmation that it's safe, @eile?